digdag-core/src/main/java/io/digdag/core/agent/OperatorManager.java

```diff
@@ -295,7 +308,13 @@ protected TaskResult callExecutor(Path projectPath, String type, TaskRequest mer
         DefaultSecretProvider secretProvider = new DefaultSecretProvider(
                 secretContext, secretAccessPolicy, grants, operatorSecretFilter, secretStore);
 
-        TaskExecutionContext taskExecutionContext = new DefaultTaskExecutionContext(secretProvider);
+        PrivilegedVariables privilegedVariables = GrantedPrivilegedVariables.build(
+                mergedRequest.getLocalConfig().getNestedOrGetEmpty("_env"),
```

Comment:
I assumed that all script processors can use environment variables. This might be wrong but as many systems do (Travis CI, Circle CI, Heroku, Docker, psql command...), using environment variable is a common way to pass credentials to a runtime. I think it is a reasonable assumption.